base/complex.jl

```diff
@@ -126,12 +126,16 @@ real(C::Type{<:Complex}) = fieldtype(C, 1)
 
 Test whether `x` or all its elements are numerically equal to some real number
 including infinities and NaNs. `isreal(x)` is true if `isequal(x, real(x))`
-is true.
+is true - this means, for `Complex` numbers, `isreal(x)` would `true` if the
+imaginary part is `0`.
```

Comment:
Kristoffer's proposal is to undo this addition here, but keep the added example below. Since that example appears in a `doctest` environment (see line 133), this "example" is actually run in continuous integration (CI) tests and compared to the "hard-coded" result/ouput below it (i.e., `false` in your case). For predicates like this, I think it is always a good idea to provide an example that yields `true` and one that yields `false`, so your addition is very helpful.